docs/list.yaml

```
@@ -50,6 +50,42 @@ PHP:
       description: Adds CORS headers support in your Laravel application
       url: https://github.com/barryvdh/laravel-cors
       tags: ['Laravel', 'CORS']
+    barryvdh/laravel-debugbar:
+      description: Laravel Debugbar (Integrates PHP Debug Bar)
+      url: https://github.com/barryvdh/laravel-debugbar
+      tags: ['Laravel', 'Debugging']
+    jenssegers/mongodb:
+      description: A MongoDB based Eloquent model and Query builder for Laravel (Moloquent)
+      url: https://github.com/jenssegers/laravel-mongodb
+      tags: ['Database', 'Moloquent']
```

1. Add other relevant tags - ex. this package is specific for Eloquent.
2. Avoid tags, that unlikely to be repeated, otherwise some time later tags cloud will be polluted with hundreds of once menitoned tags. I have doubts, that we will have any other sources, that can be marked with tag ```Moloquent```
```suggestion
      tags: ['Database', 'Eloquent']
```